teams/rustlings.toml

```diff
@@ -1,5 +1,5 @@
 name = "rustlings"
-kind = "marker-team"
+subteam-of = "lang-docs"
 
 [people]
 leads = ["shadows-withal"]
```

Comment:
Might be a good point to make me a lead since I am maintaining Rustlings alone for about 9 months now :D

Reply:
Yes, that would make sense then.  Updated.